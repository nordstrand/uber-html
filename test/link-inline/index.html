<!DOCTYPE html>
<html>
<head>
    <title>test</title>
    <link href="css/a.css" rel="stylesheet">
</head>
<body>
</body>
</html>