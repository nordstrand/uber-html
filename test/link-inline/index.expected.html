<!DOCTYPE html>
<html>
<head>
    <title>test</title>
    <style>body {
    color: blue;
}</style>
</head>
<body>
</body>
</html>